private/buf/bufcli/bufcli.go

```diff
@@ -404,7 +404,7 @@ func NewWireImageConfigReader(
 		storageosProvider,
 		NewFetchReader(logger, storageosProvider, runner, moduleResolver, moduleReader),
 		bufmodulebuild.NewModuleBucketBuilder(),
-		bufimagebuild.NewBuilder(logger, moduleReader),
+		bufimagebuild.NewBuilder(logger, moduleReader, bufmoduleref.NewModulePinResolver(clientConfig)),
```

Comment:
Can you match the style in the codebase and either have all args on one line, or all args separated by newlines?